lib/github-pages/configuration.rb

```diff
@@ -83,6 +83,10 @@ def processed(site)
         site.instance_variable_set :@_github_pages_processed, true
       end
 
+      def disable_whitelist?
+        Jekyll.env == "development" && ENV["DISABLE_WHITELIST"] == "1"
```

Comment:
What if I write `DISABLE_WHITELIST=true`? That'd fail here. Could we be more flexible on its value?